packages/apps/graph/src/devnet/simulation/simulate.ts

```diff
@@ -110,7 +141,16 @@ export async function simulate({
           counter = 0;
         }
 
-        const balance = (await getBalance(account)) as number;
+        const balance = parseFloat(
+          (
+            await getBalance(
+              account.account,
+              dotenv.NETWORK_ID,
+              account.chainId || dotenv.SIMULATE_DEFAULT_CHAIN_ID,
+              dotenv.NETWORK_HOST,
+            )
+          )?.decimal || '0',
+        );
```

Comment:
This is causing an issue. The result from `getBalance` seems to be a number and therefore the `.decimal` results in undefined, which means no transfers are done, since the balance will always result in 0.

Reply:
@javadkh2

Reply:
Fix for now: https://github.com/kadena-community/kadena.js/pull/1335/files#diff-9dec7dc344d8015e36530e9ff14865099e96a8d292205d09cd7c114b296e7e09R191